feature/platform/transceiver/zr_logical_channels_test/README.md

```diff
@@ -0,0 +1,98 @@
+# TRANSCEIVER-11: Telemetry: 400ZR Optics logical channels provisioning and related telemetry.
+
+## Summary
+
+Routing devices that support transceivers with built-in DSPs like 400ZR consume
+the [OC-terminal-device model](https://openconfig.net/projects/models/schemadocs/jstree/openconfig-terminal-device.html)
+model.
+The ZR signal in these transceivers traverses through a series of
+terminal-device/logical-channels. The series of logical-channel utilizes the
+assignment/optical-channel leaf to create the relationship to
+OPTICAL_CHANNEL. For 400ZR 1x400GE mode this heirarchy looks like:
+400GE Eth. Logical Channel => 400G Coherent Logical Channel => OPTICAL_CHANNEL
+Purpose of this test is to verify the logical channel provisioning and related
+telemetry.
+
+## Procedure
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module.
+*   To establish a point to point ZR link ensure the following:
+      * Provision the ZR modules using appropriate 
+        terminal-device(logical-channels) and
+        platforms-comonents(Optical-channel) configuration through the
+        following OC paths:
+        * /terminal-device/logical-channels/channel/config/admin-state
+        * /terminal-device/logical-channels/channel/config/description
+        * /terminal-device/logical-channels/channel/config/index
+        * /terminal-device/logical-channels/channel/config/logical-channel-type
+        * /terminal-device/logical-channels/channel/config/rate-class
+        * /terminal-device/logical-channels/channel/config/trib-protocol
+        * /terminal-device/logical-channels/channel/logical-channel-assignments/assignment/config/allocation
+        * /terminal-device/logical-channels/channel/logical-channel-assignments/assignment/config/assignment-type
+        * /terminal-device/logical-channels/channel/logical-channel-assignments/assignment/config/description
+        * /terminal-device/logical-channels/channel/logical-channel-assignments/assignment/config/index
+        * /terminal-device/logical-channels/channel/logical-channel-assignments/assignment/config/logical-channel
+        * /terminal-device/logical-channels/channel/logical-channel-assignments/assignment/config/optical-channel
+
+*   Also ensure optical channel related tunable parameters are set through the
```

Comment:
I suggest creating a sub-section for each group of paths you want to configure together.

ie:

#### TRANSCEIVER-11.1 - Test optical channel and tunable parameters
* set some paths and values
* verify

#### TRANSCEIVER-11.2 - Test Ethernet logical-channels
* set some paths and values
* verify

#### TRANSCEIVER-11.2 - Test Coherent logical-channels
* set some paths and values
* verify

This way when there are failures in the test, it is easier to trace from an error log within a specific section to the documentation.

Reply:
Done